stockAndPrices/db/sampleConnectDB.py

```diff
@@ -0,0 +1,21 @@
+from pymongo import MongoClient
+
```

Comment:
Lets move this file to the services directory, and leave the db directory for just db related files.